app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,43 @@
-<h2>Log in</h2>
 
-<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
+<div class ="container">
+  <div class="user-sessions">
+    <div class="card-signin" >
+      <div class="login-heading" style="font-weight: lighter;"> 
+      <%= image_tag("/login-icon.png", :alt => "img") %>
```

Comment:
It should work without a forwarding slash. Also, use new syntax like `alt: 'img'`.